src/Form/organisms/AreaUploadManager.tsx

```diff
@@ -148,14 +155,21 @@ const AreaUploadManager: React.FC<IAreaUploaderManager> = ({
             )
         }
 
-        <InputButtonWrapper>
+        <InputButtonWrapper hasWidth={files !== null}>
```

Comment:
And if we use `gap` then we can remove this prop.

Reply:
Done